packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -267,20 +275,19 @@ export class ApolloServerBase<Request = RequestInit> {
     if (this.engine || engineInRequestPath) this.engineEnabled = true;
   }
 
-  async request(request: Request) {
+  request(request: Request) {
```

Comment:
just want to reiterate now that 2.0 is me and you that I find the name of this method to be incomprehensible and I hope we change it before release. Doesn't have to be in this PR.

Reply:
Absolutely agree! Will be another PR most likely